### DISCUSSION TITLE
Allow bundled dashboards to be savable in app plugins

### DISCUSSION THREAD

**Discussion by alaaeelsayed** [grafana/grafana#63879] [Plugins]
Hello,

I've been working with App plugins for Grafana but I'm having trouble including dashboards that act more like a default than a forced uneditable one. Is this already possible?

**Comment by alaaeelsayed** [comment]
Bumping this, incase someone already knows a way to do this.

**Comment by alaaeelsayed** [comment]
Bump

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
